app/src/marketplace/containers/ProductController/useEditableProductActions.js

```diff
@@ -62,10 +62,14 @@ export function useEditableProductActions() {
         touch('imageUrl')
     }, [commit, touch])
     const updateImageFile = useCallback((image: File) => {
-        commit('Update image file', (p) => ({
-            ...p,
-            newImageToUpload: image,
-        }))
+        commit('Update image file', (p) => {
+            const nextProduct = {
+                ...p,
+                newImageToUpload: image,
+            }
+            delete nextProduct.imageUrl
+            return nextProduct
+        })
```

Comment:
How about

```jsx
commit('Update image file', ({ imageUrl, ...p }) => ({
    ...p,
    newImageToUpload: image,
}))
```

You wouldn't have to `delete`.

Reply:
🎉 🎉 🎉